plugins/chat/assets/javascripts/discourse/components/chat-message-actions-desktop.hbs

```diff
@@ -1,56 +1,61 @@
-<div class="chat-message-actions-container" data-id={{this.message.id}}>
+<div
+  class="chat-message-actions-container"
+  data-id={{@message.id}}
+  {{did-insert this.attachPopper}}
+  {{will-destroy this.destroyPopper}}
```

Comment:
LMK if this is not actually necessary, I just thought it was odd we had no teardown for the popper

Reply:
Also I am pretty sure `did-insert` is fine as a replacement for `didReceiveAttrs`

Reply:
Probably not necessary as it goes away with the node, but cleaner so we can keep it